data/data/kubevirt/masters/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "cluster_id" {
+  description = "The ID of Openshift cluster"
+}
+
+variable "master_count" {
+  description = "The number of master vm instances"
+}
+
+variable "namespace" {
+  type        = string
+  description = "The namespace/project in the infracluster which all the tenantcluster resources should be created in"
+}
+
+variable "storage" {
+  type        = string
+  description = "master VM disk size, of type Quantity (see: https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go)"
+}
+
+variable "memory" {
+  type        = string
+  description = "master VM memory size, of type Quantity (see: https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go)"
+}
+
+variable "cpu" {
+  type        = string
+  description = "master VM number of cores"
+}
+
+variable "ignition_data" {
+  type        = string
+  description = "Ignition config file contents of the master VMs"
+}
+
+variable "storage_class" {
+  type        = string
+  description = "The \"class\" of the storage located in the infracluster"
+}
+
+variable "network_name" {
+  type        = string
+  description = "The name of the sub network created in the infracluster which should be used by the tenantcluster resources"
+}
+
+variable "pv_access_mode" {
+  type        = string
+  description = "The access mode which all the persistant volumes should be created with [ReadWriteOnce,ReadOnlyMany,ReadWriteMany]"
```

Comment:
ReadOnlyMany doesn't make sense here, I think its not worth mentioning

Reply:
Fixed